README.md

```diff
@@ -159,6 +169,48 @@ directory to export a version-specific property:
 | tests.relevant   | List of relevant tests classes (a test class is relevant if, when executed, the JVM loads at least one of the modified classes) |
 | tests.trigger    | List of test methods that trigger (expose) the bug                                  |
 
+Export project-specific metadata
+--------------------------------
+Use `defects4j query -p <project_id> -q <field_list> [-o <output_file>] [-d] [-a]` 
+to generate a CSV file containing a set of requested metadata for each bug 
+in a project.
+
+By default, `defects4j query` returns a list of active bug IDs for a project.
+To request specific metadata, the `-q` flag should be provided with a 
+comma-separated list of variables from the list below. For example, 
+`defects4j query -p Chart -q "report.id,report.url" will provide the a list of
+all active bug IDs, along with the bug report ID and bug report URL for each.
+
+
+| Property              | Description                                                                         |
+|-----------------------|-------------------------------------------------------------------------------------|
+| bug.id                | Assigned bug IDs (included in all results)                                          |
+| project.id            | Assigned project ID                                                                 |
+| project.name          | Original repository name                                                            |
+| revision.id.buggy     | Commit hashes for the buggy version of each bug                                     |
+| revision.id.fixed     | Commit hashes for the fixed version of each bug                                     |
+| revision.buggy.date   | Date of the buggy commit for each bug                                               |
+| revision.fixed.date   | Date of the fixed commit for each bug                                               |
```

Comment:
To be consistent with the above two properties, this should be revision.date.<buggy|fixed>

Reply:
Corrected in latest commit.